templates/gc-audience-en.html

```diff
@@ -5,7 +5,7 @@
 	"altLangPage": "gc-audience-fr.html",
 	"pageType": "gcaudience",
 	"pageclass": "page-type-nav cnt-wdth-lmtd",
-	"dateModified": "2017-12-05",
+	"dateModified": "2023-07-12",
```

Comment:
Actually we didn't changed the template, we only fixed the broken Jekyll page. Idem for the other template.